hera_qm/tests/test_xrfi.py

```diff
@@ -26,22 +28,30 @@ def get_accuracy(f, rfi, verbose=VERBOSE):
         print '\t Found RFI: %1.3f\n\t False Positive: %1.3f' % (correctly_flagged, false_positive)
     return correctly_flagged, false_positive
 
+def fake_flags():
+    RFI = 10
+    SIZE = 100
```

Comment:
Does this overwrite the global variable? Maybe this should be passed as an optional variable, with the default as the global SIZE.